src/mlpack/methods/ann/layer/add_merge.hpp

```diff
@@ -226,6 +230,9 @@ class AddMerge
 
   //! Locally-stored weight object.
   OutputDataType weights;
+
+  //! Locally-stored copy visitor
+  CopyVisitor<CustomLayers...> copyVisitor;
```

Comment:
It would also be reasonable to just make a `CopyVisitor` inside the copy constructor and use it there.  I don't think it's needed to store one in the class.

Reply:
Yeah I will change it.